<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Shape_Type_Label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="Shape_Dim_Shape_Label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape Dimensions</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>331</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="Brush_Color_Label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Brush Color</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Brush_Color_ComboBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>black</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>white</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>darkGray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>gray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>lightGray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>red</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yellow</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="Pen_Cap_Style_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pen Cap Style</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Pen_Cap_Style_ComboBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>FlatCap</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SquareCap</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RoundCap</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="Brush_Style_Label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Brush Style</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Brush_Style_ComboBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>SolidPattern</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HorPattern</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VerPattern</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NoBrush</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="Pen_Join_Style_Label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pen Join Style</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Pen_Join_Style_ComboBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>MiterJoin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BevelJoin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RoundJoin</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="Pen_Style_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pen Style</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Pen_Style_ComboBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>NoPen</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SolidLine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DashLine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DotLine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DashDotLine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DashDotDotLine</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="Pen_Width_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pen Width</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Pen_Width_LineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Pen_Color_Label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pen Color</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Pen_Color_ComboBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>black</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>white</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>darkGray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>gray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>lightGray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>red</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yellow</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="perimeter_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Perimeter:</string>
      </property>
     </widget>
     <widget class="QLabel" name="area_label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Area:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="Allignment_Label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Allignment</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Allignment_ComboBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AllignLeft</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AllignRight</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AllignTop</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AllignBottom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AllignCenter</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="Allignment_Label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Text_LineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Text_Point_Size_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Text Point Size</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Text_Point_Size_LineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Font_Family_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Font Family</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Font_Family_LineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Font_Style_Label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Font Style</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Font_Style_ComboBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>StyleNormal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>StyleItalic</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>StyleOblique</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="Font_Weight_Label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Font Weight</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Font_Weight_ComboBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Thin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Light</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bold</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="Text_Color_Label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Text Color</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Text_Color_ComboBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>131</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>black</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>white</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>darkGray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>gray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>lightGray</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>red</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yellow</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="Shape_Dim_LineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Shape_Type_Combo_Box">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Line</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Polyline</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Polygon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rectangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ellipse</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Text</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Shape</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Delete_Shape_ComboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Delete_Shape_Label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit/Delete Shape</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete_shape">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_edit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuFormat"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
